### ISSUE TITLE
Remove Ansible Dependency

### ISSUE THREAD

**Issue by cookrn** [cookrn/urbit-vagrant#9]
Convert the Vagrant provisioner to shell scripts and run the playbooks locally inside the VM

**Comment by revprez** [comment]
For users without local python and Ansible?

**Comment by cookrn** [comment]
Yep. Just thinking it'd be possible to bootstrap ansible inside the VM using bash, then use the same playbooks there. That way, the user install requirements are less.

**Comment by revprez** [comment]
a :shell prov invocation to install ansible, followed by an [ansible_local](https://www.vagrantup.com/docs/provisioning/ansible_local.html) prov invocation to execute urbit.yml under /vagrant.  It's worth a shot.

**Comment by revprez** [comment]
Actually, seems ansible_local even has sufficient functionality to install ansible on the guest itself.

**Comment by cookrn** [comment]
You're right! https://www.vagrantup.com/docs/provisioning/ansible_local.html
